dali/pipeline/operator/op_schema.h

```diff
@@ -744,10 +824,24 @@ class DLL_PUBLIC OpSchema {
                  "Argument name \"" + s + "\" is reserved for internal use");
   }
 
+  inline void CheckInOutIndex(int index, bool is_input) const {
+    if (is_input) {
+      CheckInputIndex(index);
+    } else {
+      CheckOutputIndex(index);
+    }
+  }
```

Comment:
This monster should go back to the fiery chasm from whence it came ;)

Reply:
Done